rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -421,7 +421,7 @@ error_code cellHddGameCheck(ppu_thread& ppu, u32 version, vm::cptr<char> dirName
 	// 40 GB - 1 kilobyte. The reasoning is that many games take this number and multiply it by 1024, to get the amount of bytes. With 40GB exactly,
 	// this will result in an overflow, and the size would be 0, preventing the game from running. By reducing 1 kilobyte, we make sure that even
 	// after said overflow, the number would still be high enough to contain the game's data.
-	get->hddFreeSizeKB = 40 * 1024 * 1024 - 1;
+	get->hddFreeSizeKB = 40 * 1024 * 1024 - 256;
```

Comment:
The comment above should be updated. Also would be nice if this same copy paste could be merged into one variable/function.